README.md

```diff
@@ -18,11 +19,26 @@ Grep through all mocha tests for any `only` tests.
         .pipe(grepFail([ 'it.only', 'describe.only' ]));
     });
 
+Grep through a file where you expect a specific version number to appear. 
+
+    Contents of foo.js
+    ==========================
+    {
+      version: '123';
+    }
+    ==========================
+
+    gulp.task('default', function () {
+      return gulp.src('foo.js')
+        .pipe(grepFail('123', {inverse: true}));
+    });
+
 ## API
 
-`grepFail(predicates)`
+`grepFail(predicates, [options])`
 
   * **predicates** `(string|array)` string(s) to grep for and fail if they exist
+  * **options.inverse** `(boolean) - optional` default behaviour is to throw an error if a match is found. Optionally, you can set this param to true so that it fails if it is expecting values to match.
```

Comment:
Maybe we could make this clearer:

`default behavior is to throw an error if a match is found. Set `options.inverse` to `true` to throw an error if the match is *not* found.`